test/helper/routeHelper.js

```diff
@@ -57,7 +57,7 @@ async function addFileRouteLegacy(file) {
 
 async function postRunProcess(app, authToken, inputs, outputs) {
   let req = request(app)
-    .post('/run-process')
+    .post(`/${API_MAJOR_VERSION}/run-process`)
```

Comment:
These are breaking changes to the demo. Is the plan to update the demo after this?

Reply:
Yes. Going to create a ticket for the demo changes